composer.json

```diff
@@ -35,7 +35,9 @@
         "rvdv/nntp": "dev-preserve-empty-lines@dev",
         "zbateson/mail-mime-parser": "^0.4.3",
         "vlucas/phpdotenv": "^2.4",
-        "sentry/sentry": "^1.7"
+        "sentry/sentry": "^1.7",
+        "react/http": "^0.7.2",
+        "blackfire/php-sdk": "^1.8"
```

Comment:
Afair you only need this sdk when the extension is not loaded.

You could also do a extension_loaded check before registering the middleware and drop this dependency